metabase/metabase
# Mantine Tabs

Epic https://github.com/metabase/metabase/issues/32268

Adds our wrapper for [Tabs](https://v6.mantine.dev/core/tabs/).

How to verify:
- yarn storybook
- Open http://localhost:6006/?path=/docs/navigation-tabs--default-story
- Verify it matches Figma https://www.figma.com/file/uPYsD4ncNpQPFzxsLnTnGd/Navigation-%2F-Tabs?type=design&node-id=1-96&mode=design&t=dtMJ59HbOKZ8TOgJ-0

<img width="930" alt="Screenshot 2023-10-05 at 13 27 15" src="https://github.com/metabase/metabase/assets/8542534/a8aba3c1-6877-4ef6-9efd-c8fece894cb4">
